src/main/scala/beam/router/BeamSkimmer.scala

```diff
@@ -40,12 +39,15 @@ import scala.util.control.NonFatal
 
 //TODO to be validated against google api
 class BeamSkimmer @Inject()(
+  val beamServices: BeamServices,
   val beamScenario: BeamScenario,
   val geo: GeoUtils
 ) extends LazyLogging {
   import BeamSkimmer._
   import beamScenario._
 
+  def beamConfig: BeamConfig = beamServices.beamConfig
```

Comment:
There are still a number of places where `beamScenario.beamConfig` is used - I assume that should use the new `def`

Reply:
i will fix it